src/System.Net.NameResolution/tests/PalTests/project.json

```diff
@@ -24,12 +24,16 @@
     "System.Text.RegularExpressions": "4.2.0-beta-24403-05",
     "System.Threading": "4.0.12-beta-24403-05",
     "System.Threading.Tasks": "4.0.12-beta-24403-05",
+    "System.Net.Sockets": {
```

Comment:
Thanks @CIPop for the suggestion, currently testing a fix with the warning enabled. Am I right to assume that we require the SocketType and ProtocolType from the Net.Internals implementation as opposed to the Net.Sockets one?